project/scripts/dates.py

```diff
@@ -40,4 +41,32 @@ def date_to_epoch(date):
     dt = datetime(year=date.year, month=date.month, day=date.day)
     utc_time = dt.replace(tzinfo = timezone.utc) 
     utc_dt = utc_time.timestamp()
-    return int(utc_dt)
\ No newline at end of file
+    return int(utc_dt)
+
+
+def get_all_dates(initial_date):
+    """
+    Given the initial date, return a list of all dates between then
+    and now in epoch form, midnight UTC time
+    """
+    end_date = date_to_epoch(get_current_date())
+    date = initial_date
+    dates = []
+    while date <= end_date:
+        dates.append(date)
+        date += SECONDS_ONE_DAY
+    return dates
```

Comment:
is there a library to help with this, like add a day to a date, then convert to epoch, I have heard of a leap second but unsure how often it happens

Reply:
There is but I'd have to convert the epoch back to a date object and then back to an epoch which seems clunky. Also this matches how it is done at the java end so it should be consistent even in the case where there is leap second which is very unusual anyway.